woocommerce/woocommerce-ios
# Login: fix button loading indicator color in dark mode

Fixes https://github.com/woocommerce/woocommerce-ios/issues/2716
Fixes https://github.com/woocommerce/woocommerce-ios/issues/3386

- [x] ⚠️ Update pod to release version after https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/562 is merged ⚠️ 

## Why

Please see the PR description in the authenticator PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/562

## Changes

- Pointed WordPressAuthenticator pod to PR branch (will update to the release version once the PR is merged)
- Passed appropriate spinner color to the new style parameter

## Testing

- Run `bundle exec pod install --repo-update`
- Launch the app in logged out state, or log out from the app
- Tap "Continue With WordPress.com"
- Enter an email
- Tap "Continue" --> the spinner color should look the same as before
- Go back to the prologue screen
- Tap "Enter Your Store Address"
- Enter a site address
- Tap "Continue" --> the spinner color should look the same as before

## Example screenshots

\ | before | after
-- | -- | --
log in with email | ![Simulator Screen Shot - iPhone 8 Plus - 2021-01-14 at 16 50 04](https://user-images.githubusercontent.com/1945542/104574223-be204f80-5690-11eb-8280-a1ec1304becd.png) | ![Simulator Screen Shot - iPhone 8 Plus - 2021-01-14 at 17 30 06](https://user-images.githubusercontent.com/1945542/104574236-c1b3d680-5690-11eb-9deb-50a250d5b2c4.png)
log in with site address | ![Simulator Screen Shot - iPhone 8 Plus - 2021-01-14 at 16 51 14](https://user-images.githubusercontent.com/1945542/104574266-c9737b00-5690-11eb-980d-2f09d0675f1c.png) | ![Simulator Screen Shot - iPhone 8 Plus - 2021-01-14 at 17 30 37](https://user-images.githubusercontent.com/1945542/104574286-cc6e6b80-5690-11eb-905a-3febd37615a0.png)

Update release notes:
